src/sentry/data_export/tasks.py

```diff
@@ -22,7 +22,9 @@
 
 
 @instrumented_task(name="sentry.data_export.tasks.assemble_download", queue="data_export")
-def assemble_download(data_export_id, limit=1000000, environment_id=None):
+def assemble_download(
+    data_export_id, limit=1000000, batch_size=SNUBA_MAX_RESULTS, environment_id=None
```

Comment:
Is there any concern if the `SNUBA_MAX_RESULTS` change between task runs? 
nit: Also I believe `limit` here means the export limit, do you mind either changing the variable to `export_limit` or adding a comment explaining that?

Reply:
`SNUBA_MAX_RESULTS` changing shouldn't cause any issues as once the `batch_size` is set, that particular task will export a fixed batch size at time. Even once the task is chunked into multiple tasks (coming up), a different `batch_size` for each will only result in the task exporting more/less in a single batch. Also the tasks will run sequentially, so until the current task is completed, the next task will not start, at which point we know where the next batch will begin.

Reply:
Will rename the variable for clarity.

Reply:
I renamed it, and also added `**kwargs` for rolling compatibility as documented at https://develop.sentry.dev/services/queue/